.gitignore

```diff
@@ -37,6 +37,7 @@ build/Release
 
 # Dependency directories
 node_modules/
+packages/truffle/node_modules
```

Comment:
The above node_modules/ doesn't work for all directories? 

Can we use wildcards to make sure it will work across all directories?

Reply:
oh I'm big dumb

sorry usually my VS Code is configured to grey out files that are .gitignored, but for some reason it wasn't

Reply:
addressed!